README.md

```diff
@@ -16,9 +16,9 @@ This project uses deep learning computer vision to label images taken by motion-
 If you just want to get labels for your images, you can use the following steps to run a service that passes images through a trained model.
 
 1. Make sure [Docker](https://www.docker.com/get-started) is installed and running.
-2. Run `docker pull gsganden/autofocus_serve` to download the app image. (Note that it takes up about 4GB of disk space.)
-3. Run `docker run -p 8000:8000 gsganden/autofocus_serve` to start the app.
-4. Make POST requests against the app to get predictions. For instance, with the base of this repo as the working directory you can send the image `fawn.jpeg` to the app with `curl -F "file=@./gallery/fawn.jpeg" -X POST http://0.0.0.0:8000/predict`. Or you can create a zip file of the `gallery` directory and send it to the app with `curl -F "file=@gallery.zip" -X POST http://0.0.0.0:8000/predict_zip`. See `autofocus/predict/example_post.py` or `autofocus/predict/example_post.R` for example scripts that make requests using Python and R, respectively.
+2. Run `docker pull gsganden/autofocus_serve:1.2.2` to download the app image. (Note that it takes up about 4GB of disk space.)
+3. Run `docker run -p 8000:8000 gsganden/autofocus_serve:1.2.2` to start the app.
```

Comment:
I think this will be a pain to update as the project gets updated. One idea:

Instead of `autofocus/_version.py`, what if you had a file called `VERSION` which only looks like this

```
1.2.2
```

This could then be reused by `autofocus` like

```
with open("../VERSION", "r") as f:
    package_version = f.read()
```

And then these instructions would be like

```
docker run -p 8000:8000 gsganden/autofocus_serve:$(cat VERSION)
```

Consider it! Could totally be a thing you put in a separate issue to get picked up later, but I think it would be beneficial to reduce the number of duplicated places that have to be kept in sync with `_version.py`.

Reply:
I tried to do something like this for another project, and it turned into a minor debacle because VERSION wasn't available everywhere. I'm willing to give it another try because the duplication bothers me and definitely opens up possibilities for error. I think I just need to take the time to figure out manifest files and such.

Reply:
I don't see a decent way to avoid duplication in CHANGELOG.md. We could treat the version number in the top CHANGELOG.md entry as the source of truth, but pulling it out requires something like a regex, and I don't see how that would work in the docker commands. We could have some automated build process that generates VERSION from CHANGELOG.md, but that seems like more hassle than it's worth.

LMK if you have any potentially better ideas.

Reply:
Oh wait. Part of the idea of using Dockerhub was to allow someone to download the app and run it separately from the repo.

Reply:
I'm definitely open to ideas.